packages/sources/paypal/src/index.ts

```diff
@@ -2,9 +2,11 @@ import { expose } from '@chainlink/ea-bootstrap'
 import { endpointSelector, makeExecute } from './adapter'
 import { makeConfig, NAME } from './config'
 
+const adapterContext = { name: NAME }
+
```

Comment:
Since we are changing this line, could we modify this to be standard with a standard export style?
```suggestion
const { server } = expose(adapterContext, makeExecute(), undefined, endpointSelector)
```